index.js

```diff
@@ -101,7 +102,8 @@ async function terraform(terraformDirectory, args) {
   }
 
   core.startGroup('Run terraform init');
-  const tfi = await terraform(terraformDirectory, ['init']);
+  let tfi_args = !terraformBackend ? ['init'] : ['init', '-backend-config=' + terraformBackend, '-reconfigure'];
```

Comment:
I doubt `-reconfigure` is needed here? It runs in a clean environment.

Reply:
removed it 👍🏻